app/assets/javascripts/admin/addon/services/admin-sidebar-state-manager.js

```diff
@@ -1,10 +1,13 @@
+import { alias } from "@ember/object/computed";
 import Service, { service } from "@ember/service";
 import KeyValueStore from "discourse/lib/key-value-store";
-import { ADMIN_PANEL } from "discourse/lib/sidebar/panels";
+import { ADMIN_PANEL, MAIN_PANEL } from "discourse/lib/sidebar/panels";
 
 export default class AdminSidebarStateManager extends Service {
   @service sidebarState;
   @service currentUser;
+  @alias("sidebarState.currentUserUsingAdminSidebar")
+  currentUserUsingAdminSidebar;
```

Comment:
Does this only work for reading? Asking because it looks like there's an opportunity to give `this.sidebarState.isForcingAdminSidebar` the same treatment otherwise. 🙂 

It does sound like it should work from documentation:

> Calls to `get` or `set` this property behave as though they were called on the original property.